File: BackEnd/FoodOrder.Core/ViewModels/Ratings/RatingVM.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace FoodOrder.Core.ViewModels.Ratings
{
    public class RatingVM
    {
        public int OrderID { get; set; }
        
        public int FoodID { get; set; }
        public int Star { get; set; }
        public string Comment { get; set; }
        public DateTime TimeCreate { get; set; }
        public string UserFullName { get; set; }
        public string UserID { get; set; }
    }
}

File: FrontEnd/food_order/lib/view_models/ratings/RatingVM.dart
import 'package:json_annotation/json_annotation.dart';

part 'RatingVM.g.dart';

@JsonSerializable()
class RatingVM {
  RatingVM();

  int orderID = 0;
  String userID = "";
  int foodID = 0;
  int star = 0;
  String comment = "";
  String userFullName = "";
  DateTime timeCreate = DateTime.now();

  factory RatingVM.fromJson(Map<String, dynamic> json) =>
      _$RatingVMFromJson(json);
  Map<String, dynamic> toJson() => _$RatingVMToJson(this);
}
